thoth/storages/graph/dgraph.py

```diff
@@ -2029,6 +2053,67 @@ def sync_analysis_result(self, document: dict) -> None:
         self._python_sync_analysis_result(package_extract_run, document, environment)
         self._python_file_digests_sync_analysis_result(package_extract_run, document)
 
+    @enable_vertex_cache
+    def sync_package_analysis_result(self, document: dict) -> None:
+        """Sync the given package analysis result to the graph database."""
+        package_analysis_document_id = PackageAnalysisResultsStore.get_document_id(document)
+        package_name = document["metadata"]["arguments"]["python"]["package_name"]
+        package_version = document["metadata"]["arguments"]["python"]["package_version"]
+        index_url = document["metadata"]["arguments"]["python"]["index_url"]
+
+        # TODO: capture errors on package analysis? result of package-analyzer should be a JSON with error flag
```

Comment:
Are there some flags to state that?

Reply:
@fridex I saw such TODO on package-extract, then just copied, should I remove it?